renderer/reducers/wallet.js

```diff
@@ -61,6 +61,29 @@ export const putWallet = wallet => async dispatch => {
 export const showDeleteWalletDialog = () => ({ type: OPEN_DELETE_WALLET_DIALOG })
 export const hideDeleteWalletDialog = () => ({ type: CLOSE_DELETE_WALLET_DIALOG })
 
+/**
+ * Remove a wallet entry.
+ * @param  {Object} wallet Wallet entry.
+ * @return {Promise}
+ */
+export const removeWallet = wallet => async dispatch => {
+  // Delete the wallet from the filesystem.
+  if (wallet.type === 'local') {
+    const { chain, network, wallet: walletName } = wallet
+    await window.Zap.deleteLocalWallet({ chain, network, wallet: walletName })
+  }
+
+  // Delete the wallet from the database.
+  await window.db.wallets.delete(wallet.id)
+
+  // Refresh the wallets state data.
+  await dispatch(getWallets())
+}
+
+/**
+ * Handle wallet delete action.
```

Comment:
Let's outline that `deleteWallet` is a response to a user input type of action, so it's clear what is the distinction between this and `removeWallet` 
Something like `Handle delete wallet confirmation by a user`

Reply:
Updated